styles.css

```diff
@@ -0,0 +1,168 @@
+/* global styles */
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-weight: 400;
+  font-family: "Nunito Sans", sans-serif;
+  --brand-primary: #0768ac;
+  --brand-secondary: #03c180;
+  --bg_default: #ffffff;
+  --bg_body: #f0f9ff;
+  --lines-color: #dddddd;
+  --body-text: #333333;
+  --heart-color: #dc143c;
+}
+
+input:focus,
+select:focus {
+  outline: none;
+}
+
+/* Header Styles */
+header {
+  display: flex;
+  justify-content: space-between;
+  background-color: var(--bg_default);
+  padding: 15px 50px;
+}
+
+header .logo {
+  font-size: 14px;
+  color: #0768ac;
+  font-weight: 700;
+  padding-top: 7px;
+}
+
+header .btn-wrapper {
+  display: flex;
+  justify-content: space-between;
+  gap: 4px;
+}
+
+header .btn-wrapper button {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 5px;
+  cursor: pointer;
+  background-color: var(--bg_default);
+  border-radius: 5px;
+  padding: 5px 10px;
+  border: 1.5px solid var(--lines-color);
+}
+
+header .btn-wrapper button .icon {
+  width: 15px;
+  height: 15px;
+}
+
+/* Page Heading Section Styles */
+.page-heading-container {
+  height: 110px;
+  border-top: 1px solid var(--brand-primary);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.page-heading-container .page-heading-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: start;
+  margin-top: 25px;
+}
+
+.page-heading-container .main-title {
+  font-size: 18px;
+  font-weight: 700;
+  color: var(--brand-secondary);
+}
+
+.page-heading-container .subtitle {
+  font-size: 14px;
+  color: var(--body-text);
+}
+
+/* Main Section Styles */
+main {
+  background-color: var(--bg_body);
+  width: auto;
+  height: 100vh;
+  padding: 20px 50px;
+}
+
+main .display-control {
+  display: flex;
+  align-items: center;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+  background-color: var(--bg_default);
+  padding-left: 10px;
+  border-radius: 10px;
+  height: 50px;
```

Comment:
I recommend not to set a fixed height for containers. containers should take it's its height from the content. 

in this case you have an input field and 2 select elements. but the height on each of them. this will be better because you want all inputs in the website have the same size. or few defined sizes like (sm, md, lg).